src/screens/DiscoveryScreenV2/elements/RenderItem.js

```diff
@@ -132,9 +132,9 @@ const styles = StyleSheet.create({
   container: {
     marginTop: 8,
     borderRadius: SIZES.radius,
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
     elevation: 0.0,
-    borderColor: COLORS.blackgrey,
+    borderColor: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `backgroundColor` and `borderColor` properties seem fine as they are just color updates. However, please ensure that these new colors (`COLORS.almostBlack` and `COLORS.gray400`) provide sufficient contrast for accessibility purposes. Also, make sure that these color changes align with the overall theme of your application.

```diff
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
     elevation: 0.0,
-    borderColor: COLORS.blackgrey,
+    borderColor: COLORS.gray400,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->